wandb/docker/__init__.py

```diff
@@ -14,7 +17,7 @@
 log = logging.getLogger(__name__)
 
 
-def shell(cmd):
+def shell(cmd: List[str]) -> Optional[str]:
```

Comment:
This whole file deserves some additional cleanup but it's basically a thin wrapper ontop of the `docker` command.